ietf/liaisons/views.py

```diff
@@ -525,3 +526,17 @@ def liaison_resend(request, object_id):
     messages.success(request,'Liaison Statement resent')
     return redirect('ietf.liaisons.views.liaison_list')
 
+
+@role_required("Secretariat", "IAB", "Liaison Coordinator", "Liaison Manager")
+def list_other_sdo(request):
+    def _sdo_order_key(obj:Group)-> Tuple[str,str]:
```

Comment:
nit: better just to do `-> tuple[str, str]` (lower-case `t`, no import of `Tuple`)